Components/Navigation/index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
+    />
+
+    <link rel="stylesheet" href="../../index.css" />
+
+    <title>Navigation</title>
+  </head>
+  <body>
+    <div id="kash-container" class="kash-container-root">
```

Comment:
I didn't understand the need of the word 'kash' here, just a suggestion to use short meaningful id/class names like 'container' or 'container-root' throughout the code.

Reply:
kash is the short name of library here so if it used in any other app that do not overlap the names so i used this i discussed with my team leader he said it's ok to use and should be meaning ful